packages/jx.yml

```diff
@@ -1,3 +1,4 @@
-version: 2.0.404
+version: 2.0.537
```

Comment:
I'd keep the minimum at `2.0.404` - our own builds are using older versions, most users are going to be on older versions, etc.